src/documents/checks.py

```diff
@@ -69,3 +70,17 @@ def parser_check(app_configs, **kwargs):
         ]
     else:
         return []
+
+
+@register()
+def filename_format_check(app_configs, **kwargs):
+    if settings.FILENAME_FORMAT:
+        converted_format = convert_to_django_template_format(settings.FILENAME_FORMAT)
+        if converted_format != settings.FILENAME_FORMAT:
+            return [
+                Error(
```

Comment:
Also, curious about why this is a hard-stop? Is that basically necessary? I worry a lot of people will get snagged by this.

Reply:
Yeah, meant to be a warning

Reply:
Downgraded this to a warning